gui/src/components/ProcessList.svelte

```diff
@@ -11,7 +11,7 @@
     deleteProcess,
   } from '../lib/process/store';
   import { setLogVisibility, visibleLogsStore } from '../lib/logs/visible-logs';
-  import type { ProcessDescription } from '../lib/process/types';
+  import type { ComponentDetails, } from '../lib/process/types';
```

Comment:
prettier should be running on svelte files

Reply:
Ah, my bad. Set up correctly now.